ocaml/xapi/xapi_globs.ml

```diff
@@ -1803,6 +1805,10 @@ module Resources = struct
       , trace_log_dir
       , "Directory for storing traces exported to logs"
       )
+    ; ( "pool-recommendations-dir"
+      , pool_recommendations_dir
+      , "Pool recommendations directory"
```

Comment:
I would say: "Directory containing files with recommendations in key=value format."